### DISCUSSION TITLE
Windows shutdown dialog

### DISCUSSION THREAD

**Discussion by Wizzzzzard** [Open-Shell/Open-Shell-Menu#1603] [Q&A] [answered]
Is it possible to remove the extra popup shutdown dialog that appears after I click shutdown? I don't want to have to click shutdown twice.

For years I have clicked shutdown and walked away while my PC shuts down but now because of this issue I often click shutdown and walk away but find later that my computer is still on and displaying the shutdown dialog window.

I like the benefits of open-shell but this is ruining the experience for me.

**Comment by ge0rdi** [comment]
What dialog you mean? Can you post a screenshot?

**Comment by Wizzzzzard** [comment]
![Shutdown Dialog](https://github.com/Open-Shell/Open-Shell-Menu/assets/30133795/4ac7f0b6-78ff-422d-aa2b-a106c85ddf23)

**Comment by ge0rdi** [comment, marked as answer]
It seems you are using `Classic` style for start menu.
There the `Shut down...`  button is meant to display shutdown dialog.

You can see it in settings `Customize Start Menu` there is `Shutdown Dialog` item.
If you want button that will do shutdown directly you can drag `Shutdown` item from the right column and place it into left one.

Or you can use the _arrow_ on the right side of `Shut Down...` button to select actual power action.

Or you can use `Windows 7 style` for start menu.
Here button does selected action (you can configure it) while you can still select different action in arrow menu on the right side.

**Comment by Wizzzzzard** [comment]
I dragged the shutdown item :)

Thanks for the info.
